app/pandas.ipynb

```diff
@@ -458,36 +1195,78 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "s = pd.Series([\"numpy\", \"pandas\", \"matplotlib\"])"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-23T17:27:38.913196Z",
+     "start_time": "2024-10-23T17:27:38.908672Z"
+    }
+   },
+   "outputs": [],
+   "execution_count": 50
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# make all words in uppercase"
+    "s = s.apply(lambda x: x.upper())\n",
+    "s"
```

Comment:
Ensure that the operation to convert words to uppercase does not modify the original Series 's'. Consider creating a new Series or using a method that doesn't alter 's' directly.